command/oauth/cmd.go

```diff
@@ -157,8 +163,18 @@ $ step oauth --client-id my-client-id --client-secret my-client-secret \
 				Usage: "Email to authenticate",
 			},
 			cli.BoolFlag{
-				Name:  "console, c",
-				Usage: "Complete the flow while remaining only inside the terminal",
+				Name: "console, c",
+				Usage: `Complete the flow while remaining only inside the terminal.
+
+NOTE: This flag instructs the CLI to retrieve a token using the Out Of Band flow,
+which has been deprecated. In an upcoming release this flag will be updated to
+use the Device Authorization Grant flow (https://datatracker.ietf.org/doc/html/rfc8628#section-3.2).
+Please use the '--device' flag to use the new flow in the interim.`,
```

Comment:
I don't think we should change the way `--console` works for the moment because it will break people's existing implementations. Once Google formally EOL's the OOB flow (sometime in August) then we can switch `--console` over to be the same as `--device`.

I agree that the `device` flow is better supported, but we shouldn't immediately break people's existing flow by changing how our OOB flow works. We can at least wait until Google breaks it for us.

Reply:
Can you add an issue for this and I will resolve this?

Reply:
Another option would be to do the change now for some idPs that we know that doesn't support oob, e.g. okta

Reply:
Add an issue for what? Moving `--console` to the `--device` implementation once the flow is completely deprecated?

Reply:
Yes, and it won't break the flow for people not using Google, and that's easy to detect.